apps/playground/src/app/components/tabs/wizard/wizard-dropdown-demo-modal.component.ts

```diff
@@ -0,0 +1,59 @@
+import { Component, OnInit } from '@angular/core';
+import { FormBuilder, FormGroup } from '@angular/forms';
+import { SkyModalInstance } from '@skyux/modals';
+
+@Component({
+  selector: 'app-wizard-dropdown-demo-modal',
+  templateUrl: './wizard-dropdown-demo-modal.component.html',
+})
+export class WizardDropdownDemoModalComponent implements OnInit {
+  public myForm: FormGroup;
+  public title = 'Wizard Tabset Example';
+  public activeIndex = 0;
+  public step2Disabled = true;
+  public step3Disabled = true;
+  public saveDisabled = true;
+
+  constructor(
+    public instance: SkyModalInstance,
+    private formBuilder: FormBuilder
+  ) {}
+
+  public ngOnInit(): void {
+    this.myForm = this.formBuilder.group({
+      requiredValue1: undefined,
+      requiredValue2: false,
+      requiredValue3: undefined,
+    });
+
+    this.myForm.valueChanges.subscribe(() => {
+      this.checkRequirementsMet();
+    });
+  }
+
+  public checkRequirementsMet(): void {
+    const requirement1Met = this.myForm.get('requiredValue1').value;
```

Comment:
Totally optional, but I think (and the code like it in other files) this would benefit from eliminating the local `const`s here and assigning the values directly to the public step disabled properties.

```
this.step2Disabled = !this.myForm.get('requiredValue1').value;
this.step3Disabled = !this.myForm.get('requiredValue2').value;
this.saveDisabled = !this.myForm.get('requiredValue3').value;
```